app/Makefile

```diff
@@ -92,6 +92,9 @@ build:
 start:
 	docker compose up --detach
 
+start-debug:
+	docker compose -f ../docker-compose.yml -f ../docker-compose.debug.yml up --detach
+
```

Comment:
debug file gets launched second to be more clear that it provides the latter override